### ISSUE TITLE
bash-completion gives error "__git_reassemble_comp_words_by_ref: command not found"

### ISSUE THREAD

**Issue by gzm55** [apache/maven-mvnd#670]
when enable bash completion, it fails with error "__git_reassemble_comp_words_by_ref: command not found"

```bash
bash-5.1$ source bin/mvnd-bash-completion.bash
bash-5.1$ PATH+=`pwd`/bin
bash-5.1$ mvnd bash: __git_reassemble_comp_words_by_ref: command not found
bash: words_: bad array subscript
```

**Comment by gnodet** [comment]
You need to install [git bash completion](https://stackoverflow.com/questions/12399002/how-to-configure-git-bash-command-line-completion).

**Comment by gzm55** [comment]
can we inline this implicit dependency shell function?

**Comment by gnodet** [comment]
#gzm55  can you check the fix at #682 ?

**Comment by gzm55** [comment]
> @gzm55 can you check the fix at #682 ?

it works @gnodet
